scrapy/Fetch/__init__.py

```diff
@@ -0,0 +1,46 @@
+import asyncio
+import logging
+import scrapy
+import warnings
+
+from twisted.internet.defer import Deferred
+
+from scrapy.core import config
+from scrapy.core import engine
+from scrapy.utils.log import logformatter_adapter, failure_to_exc_info
+from scrapy import signals
+from scrapy.core.downloader import Downloader 
+
+
+
+logger = logging.getLogger(__name__)
+
+
+
+class Fetch():
```

Comment:
I think we should make Request class to be awaitable, not create a separate Fetch class.

Reply:
Regarding awaiting Request class, I was having circular import issues.